common/src/Model/GameImpl.cpp

```diff
@@ -302,15 +303,14 @@ namespace TrenchBroom {
         std::vector<IO::Path> GameImpl::doExtractTextureCollections(const AttributableNode& node) const {
             const auto& property = m_config.textureConfig().attribute;
             if (property.empty()) {
-                return std::vector<IO::Path>(0);
+                return {};
             }
 
-            const auto& pathsValue = node.attribute(property);
-            if (pathsValue.empty()) {
-                return std::vector<IO::Path>(0);
+            if (const auto* pathsValue = node.entity().attribute(property)) {
```

Comment:
You need to add `; pathsValue != nullptr && !pathsValue.empty()` to the if condition match the original code.

I prefer early return style..

Reply:
The behavior is the same because `kdl::str_split` returns an empty array for an empty string. But I have changed it to early return because we used the same style just a few lines above.